ambry-network/src/main/java/com/github/ambry/network/NettyServerRequestResponseChannel.java

```diff
@@ -117,6 +128,24 @@ public List<NetworkRequest> getDroppedRequests() throws InterruptedException {
   public void shutdown() {
     networkRequestQueue.close();
   }
+
+  /**
+   * Reject request with backoff error code.
+   * @param networkRequest incoming request
+   */
+  void rejectRequest(NetworkRequest networkRequest) throws InterruptedException {
```

Comment:
this is all good, but we need some metric here to keep track of how many requests are rejected in the channel.